### PR TITLE
Deploy your application to StarlingX using FluxCD

### PR DESCRIPTION
**Pull request by Danmcaires**

### CODE REVIEW

**Review by ildikov — COMMENTED** [review]
Hi Daniel,

Thank you for submitting your blog post, and your patience on having it reviewed.

I added a few suggestions in line to further polish the text. I also set the publish date to next Tuesday, but we can move that further out if you need more time to go through review comments.

Please let me know if either comment in line needs clarification.

Thanks and Best Regards,
Ildikó

File: src/pages/blog/starlingx-deploying-your-app-2.md [hunk 1]
```diff
@@ -0,0 +1,268 @@
...
+[Kubernetes cluster(s) that StarlingX manages](https://docs.starlingx.io/operations/k8s_cluster.html)
+:
+
+- [with plain Kubernetes](https://kubernetes.io/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro/) methods;
+- with [Helm](https://helm.sh/docs/intro/using_helm/#helm-install-installing-a-package);
+- with [Flux](https://fluxcd.io/); and finally
+- as a StarlingX Application, which benefits from tight integration with the
+  [StarlingX system](https://opendev.org/starlingx/config).
+
+In the first installment of this series, you learned how to deploy an application
```
[the remark this answers is not in the document]
Comment (Danmcaires) [reply src/pages/blog/starlingx-deploying-your-app-2.md:20]: Done

File: src/pages/blog/starlingx-deploying-your-app-2.md [hunk 2]
```diff
@@ -0,0 +1,268 @@
...
+~$ kubectl get gitrepositories
+
+NAME            URL                                                           AGE   READY
+poc-starlingx   https://github.com/bmuniz-daitan/poc-starlingx-messages.git   49s   True
+```
+
+#### Helm Controller
+
+The Helm Controller is an operator that allows the management of Helm Chart
+releases in a declarative way. The resource HelmRelease will define the desired
```
[the remark this answers is not in the document]
Comment (Danmcaires) [reply src/pages/blog/starlingx-deploying-your-app-2.md:113]: Done

**Review by ildikov — APPROVED** [review]
Thanks @Danmcaires for the quick fixes. I fixed a few typos that remained in the article, so it is now ready to merge.

Thank you again for contributing, I'm looking forward to the next one in the series!

Best Regards,
Ildikó

**Pull request merged**